packages/babel-plugin-transform-typescript/test/fixtures/imports/import=-declaration/plugin.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function({ types: t }) {
+  return {
+    visitor: {
+      Identifier(path) {
+        if (path.node.name === "foo" && !path.scope.getBinding(path.node.name))
```

Comment:
If we are visiting the `foo` identifier, we can assert that the scope has registered binding:
```js
expect(path.scope.hasBinding(path.node.name)).toBe(true);
```
At the end of the traversal, we can assert that the assertion must have been tested:
```js
{
  visitor: {
    Program: {
      exit() {
        expect.assertions(1)
      }
   }
}
```
So that if the structure of `input.js` is changed, this test will not silently pass because the identifier visitor is not invoked.